example-playbooks/replace_node/replace_node.yml

```diff
@@ -24,6 +24,51 @@
       when: hostvars[new_node]['broadcast_address'] in scylla_seeds or new_node in scylla_seeds
       run_once: true
 
+    - name: Stop scylla-server on the node being replaced
+      block:
+        - name: Disable Scylla service
+          service:
+            name: scylla-server
+            enabled:  false
+          become: true
+
+        - name: Run nodetool drain
+          async_task:
+            shell: |
+              nodetool drain
+            alias: drain_scylla
+            async: 60
+            retries: 2
+            delay: 30
+          ignore_errors: true
+          register: _drain_scylla_output
+
+        - name: nodetool drain logs
+          debug: var=_drain_scylla_output
```

Comment:
nodetool drain doesn't print much AFAIR. Why is this needed?

Reply:
Logs removed